app/views/case_contacts/form/details.html.erb

```diff
@@ -54,7 +54,6 @@
           </h3>
           <% min_date = CaseContact::MINIMUM_DATE %>
           <% current_date = Time.zone.today %>
-          <% initial_value = @case_contact.occurred_at&.to_date %>
```

Comment:
unused (warning)

Reply:
it's an erb, so *probably* used by js somewhere?

Reply:
No, but I only know that because I wrote it haha. I left this straggler.